dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -1711,10 +1717,14 @@ public ProgramOutput startServerWithArgs(boolean preClean, boolean cleanStart,
 
             startedWithJavaSecurity = bootstrapHasJava2SecProps;
             if (bootstrapHasJava2SecProps) {
-                if (info.majorVersion() >= 18) {
-                    // If we are running on Java 18+, then we need to explicitly enable the security manager
+                if (javaInfo.majorVersion() >= 18 && javaInfo.majorVersion() <= 23) {
```

Comment:
Not sure why you switched to `javaInfo` instead of using `info`.  `info` is for the server's Java version instead of the test side Java version.

Reply:
That was a copy/paste error from LibertyClient.  Thanks for catching it!